kedro/template/{{ cookiecutter.repo_name }}/kedro_cli.py

```diff
@@ -121,8 +121,13 @@ def test(args):
 
 
 @cli.command()
+@click.option("--conda")
```

Comment:
```suggestion
@click.option("--conda", is_flag=True, multiple=False, help=CONDA_FLAG_HELP)
```
And make sure you add a nice helpful message at the top of the file for `CONDA_FLAG_HELP`, similar to the other ones around there.